src/Uno.UI.Tests/DependencyProperty/Given_DependencyProperty.Propagation.cs

```diff
@@ -476,8 +476,6 @@ public void When_ControlTemplate_And_Animation()
 			});
 
 			SUT.Template = template;
-
```

Comment:
Same thing here : the behavior tested here is not valid on windows, we should not enforce that in tests

Reply:
Oups here as we set the template, you're right, it should work as is